src/pumice_db.c

```diff
@@ -941,10 +949,50 @@ pmdb_sm_handler_client_write(struct raft_net_client_request_handle *rncr)
      * of ID_ANY_64bit means the object has previously attempted a write but
      * that write did not yet (or ever) commit.
      */
-    if (pmdb_req->pmdbrm_write_seqno <= obj.pmdb_obj_commit_seqno &&
+    if (pmdb_req->pmdbrm_write_seqno < obj.pmdb_obj_commit_seqno &&
         obj.pmdb_obj_commit_seqno != RAFT_ENTRY_IDX_ANY)
     {
         raft_client_net_request_handle_error_set(rncr, -EALREADY, 0, 0);
+        SIMPLE_LOG_MSG(LL_WARN, "less rncui");
+    }
+    else if (pmdb_req->pmdbrm_write_seqno == obj.pmdb_obj_commit_seqno &&
+        obj.pmdb_obj_commit_seqno != RAFT_ENTRY_IDX_ANY)
+    {
+        // pmdbApi->pmdb_fill_reply callback
+        if (pmdbApi->pmdb_fill_reply) {
```

Comment:
Please conform the formatting 'standards'  in this repo where conditional braces are on newlines:
```
if (pmdbApi->pmdb_fill_reply) 
{
...
}
```